wizeline/WLEmptyState
# Fix error loading bundle images

#### What does this PR do? Any background context you want to provide?
Images were loading from a different resource, so they weren't being displayed.
This PR also adds a xcassets file to manage the resources.
#### Where should the reviewer start?
* `WLEmptyState/Classes/WLEmptyStateDataSource.swift`
#### How should this be manually tested?
* Run the example app, in the WLEmptyState you should be able to see the empty symbol image.
#### What are the relevant tickets?
N/A
#### Screenshots (if appropriate)
![simulator screen shot - iphone 6 - 2019-02-09 at 13 15 57](https://user-images.githubusercontent.com/6756995/52525290-e44f4080-2c6c-11e9-9737-b3b42c37e740.png)

#### Questions
N/A
